ui/components/app/asset-list-item/asset-list-item.js

```diff
@@ -121,21 +122,23 @@ const AssetListItem = ({
       subtitle={secondary ? <h3 title={secondary}>{secondary}</h3> : null}
       onClick={onClick}
       icon={
-        <Identicon
-          className={iconClassName}
-          diameter={32}
-          address={tokenAddress}
-          image={tokenImage}
-          alt={`${primary} ${tokenSymbol}`}
-          imageBorder={identiconBorder}
-        />
+        // <Identicon
```

Comment:
- For now, we should still use identicon for other imported tokens
- You can test it by importing `0x55d398326f99059ff775485246999027b3197955`

Reply:
fixed in 033e5c4473853fa5d9a614cabc42898195c51bd6